lib/bundler/cli/remove.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Bundler
+  class CLI::Remove
+    def initialize(gems, options)
+      @gems = gems
+      @options = options
+    end
+
+    def run
+      # Raise error if no gems are specified
```

Comment:
comment is necessary -- it's just repeating what the line of code below is saying.

in general, comments should explain the _why_ or _how_ rather that the _what_, since that's what the code does :)